src/api.js

```diff
@@ -23,7 +35,13 @@ var API = (function() {
     address: '459-455 Boulevard de la Carrière\nGatineau\nQuébec\nJ8Y 6V7',
     maritalStatus: 'Widowed',
     children: '5',
-    income: '$15,000',
+    income: {
+      employerName: 'Aplha Corp',
+      year: 2019,
+      box12: '254528961',
+      box14: 50000,
```

Comment:
I know it's a small thing, but I think we should keep the income less than 40000 since that's supposed to be our threshold.

Reply:
Nod, in the future we might want to have those to do our error checking or something, as in, do you have too much income for this tool.